polkadot/runtime/parachains/src/assigner_on_demand/tests.rs

```diff
@@ -342,8 +359,8 @@ fn pop_assignment_for_core_works() {
 
 		// Add enough assignments to the order queue.
 		for _ in 0..2 {
-			place_order(para_a);
-			place_order(para_b);
+			place_order_run_to_101(para_a);
```

Comment:
Why do we need to trigger a session change?

Reply:
I'm not sure what the original idea was as this code was here before me. I've just checked and the test passes without this change.